spring-cloud-stream-schema/src/main/java/org/springframework/cloud/stream/schema/avro/AvroSchemaMessageConverter.java

```diff
@@ -43,28 +43,31 @@
 	/**
 	 * Create a {@link AvroSchemaMessageConverter}. Uses the default {@link MimeType} of
 	 * {@code "application/avro"}.
+	 * @param manager for schema management
 	 */
-	public AvroSchemaMessageConverter() {
-		super(new MimeType("application", "avro"));
+	public AvroSchemaMessageConverter(AvroSchemaServiceManager manager) {
+		super(new MimeType("application", "avro"), manager);
 	}
 
 	/**
 	 * Create a {@link AvroSchemaMessageConverter}. The converter will be used for the
 	 * provided {@link MimeType}.
 	 * @param supportedMimeType mime type to be supported by
 	 * {@link AvroSchemaMessageConverter}
+	 * @param manager for schema management
 	 */
-	public AvroSchemaMessageConverter(MimeType supportedMimeType) {
-		super(supportedMimeType);
```

Comment:
The same comment applies here as the public constructor above.

Reply:
added back constructor as suggested